concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1125,6 +1113,27 @@ jobs:
         - get: bosh-init-state
         - get: ssh-private-key
 
+        # task to work around potential bosh state corruption on build re-runs - 184281154
```

Comment:
I'd suggest making this comment future proof - Imagine an engineer comes along in a year and for some reason we don't have pivotal anymore, what would be a helpful comment?

Reply:
I don't see that as a problem. I believe that the comment before the Pivotal Tracker reference conveys sufficient information so as to mitigate the possibility of Pivotal Tracker becoming unavailable. I could get rid of the story number if that is likely to be confusing but it also appears in the PR name which would likely lead to a similar problem should Pivotal Tracker become unavailable. What do you think?

 If you disagree with my reasoning of course I'll change it.

Reply:
you're right 👍